js/layouts/tab-container/regular-tabs.js

```diff
@@ -124,5 +124,15 @@ $.widget( "itop.regulartabs", $.ui.tabs, {
 			this._off( prevPanels.not( this.panels ) );
 		}
 	},
-
-});
\ No newline at end of file
+	// JQuery UI overload
+	disable: function(index){
+		let panel = this._getPanelForTab( index );
+		panel.css({display: 'none'});
```

Comment:
We rather use `.show()` / `.hide()` methods directly than CSS properties

```suggestion
		panel.hide();
```